testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -1435,7 +1777,7 @@ impl BitcoinRPCRequest {
             }
         };
 
-        Ok(vec![])
+        Ok(UTXOSet { bhh, utxos })
     }
 
     pub fn send_raw_transaction(config: &Config, tx: String) -> RPCResult<()> {
```

Comment:
I think we're at the point where the transaction fee estimation logic is complex enough that it should probably have its own unit tests.  Namely, the code should have tests to ensure that:

* The RBF fee rate never decreases over successive replacements, and always increases by at least the minimal amount.
* The UTXO-consumption code can handle the case where UTXOs get invalidated by one or more arriving Bitcoin blocks (a "flash block").  It should stop attempting to send the block-commit, even if it's still pending in the mempool.
* The UTXO-consumption code can handle the case where UTXOs get invalidated by intermittent transactions.  For example, suppose my sends a block-commit, then spend its change output with another wallet outside the node.  The node should detect this and re-calculate its UTXO set.
* The fee estimation logic should strictly be based on the length of the generated transaction, including all the UTXOs it includes.
* The code should support an overall max transaction fee a block-commit can have.  The code should pick its UTXOs accordingly, so that it can always RBF its pending block-commits up to this maximum fee.  I don't think the code should attempt to send multiple, different block-commits for the same sortition.